tests/deepsparse/loggers/metric_functions/helpers/test_config_generation.py

```diff
@@ -93,6 +95,7 @@
   pipeline_outputs.labels:
   - func: predicted_classes
     frequency: 10
+  pipeline_outputs.scores:
```

Comment:
This happens because of the changes in `src/deepsparse/loggers/metric_functions/built_ins.py` above